x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/analysis/Verifier.java

```diff
@@ -70,11 +74,11 @@ public Verifier(Metrics metrics) {
     }
 
     public Map<Node<?>, String> verifyFailures(LogicalPlan plan) {
-        Collection<Failure> failures = verify(plan);
+        Collection<Failure> failures = verify(plan, x -> Collections.emptySet());
```

Comment:
Why use an emptySet when verifying failures vs verify?

Reply:
The function doesn't seem to be used anywhere, I assumed it's a debugging left-over. I've removed it now as dead code.